packages/calcite-components/src/components/stepper-item/stepper-item.tsx

```diff
@@ -335,6 +358,34 @@ export class StepperItem implements InteractiveComponent, LocalizedComponent, Lo
     );
   }
 
+  private renderAction(position: Position): VNode {
+    const isPositionStart = position === "start";
+    const path = isPositionStart ? "chevron-left" : "chevron-right";
+    const { itemPosition, totalItems, responsiveMode, layout, iconFlipRtl } = this;
+
+    return layout === "horizontal" && responsiveMode ? (
+      <calcite-action
+        appearance="transparent"
+        class={{
+          [CSS.actionIcon]: true,
+          [CSS.actionIconStart]: isPositionStart,
+          [CSS.actionIconEnd]: !isPositionStart,
+        }}
+        disabled={
+          (itemPosition === 0 && isPositionStart) ||
+          (itemPosition === totalItems - 1 && !isPositionStart)
+        }
+        icon={path}
+        iconFlipRtl={iconFlipRtl}
+        // eslint-disable-next-line react/jsx-no-bind
+        onClick={(event) => this.handleActionClick(event, position)}
+        onKeyDown={(event) => this.handleActionKeyDown(event, position)}
+        scale={this.scale}
+        text={isPositionStart ? "Previous Step" : "Next Step"}
```

Comment:
This should be in the messages property right?

Reply:
Yup, will create a separate PR for this.

Reply:
Installed PR #7918